gammapy/cube/tests/test_sherpa_.py

```diff
@@ -123,6 +126,54 @@ def testCombinedModel3DInt():
 
     assert_allclose(result.parvals, reference, rtol=1E-5)
 
+    # Add a region to exclude in the fit: Here we will exclude some events from the Crab since there is no region to
+    # exclude in the FOV for this example. It's just an example to show how it works and how to proceed in the fit.
+    # First you define the coord of your cube that you want to use for the fit
+    center_excluded_region = SkyCoord(83.60, 21.88, unit='deg')
```

Comment:
Could you use a `SkyCircleRegion` (http://astropy-regions.readthedocs.io/en/latest/api/regions.CircleSkyRegion.html#regions.CircleSkyRegion) object here? Or in best case pre-compute the exclusion mask and add it to the test data in `gammapy-extra`.